### PR TITLE
reset action collapse on render

### PR DESCRIPTION
**Pull request by josephahn**
https://github.com/marionettejs/marionette.inspector/issues/244

### CODE REVIEW

File: extension/js/inspector/app/modules/Activity/models/ActionCollection.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  var ActionCollection = Backbone.Collection.extend({
+
+    model: Action,
+
+    resetCollapse: function() {
+      var actions = this.models;
+      _.each(actions, function(action) {
+        action.isCollapsed = true;
+      });
```
Comment (jasonLaster) [inline extension/js/inspector/app/modules/Activity/models/ActionCollection.js]: small trick

```js
this.each(function(action) { action.isCollapsed = true});
```

**Comment by jasonLaster** [comment]
nice!

**Comment by samccone** [comment]
:clap: @josephahn

**Comment by josephahn** [comment]
:facepunch: first bump
:hand: high five
:+1: thumbs up
@samccone @jasonLaster

**Comment by jasonLaster** [comment]
:dancers: :fireworks: :dancers:

**Pull request merged**